src/jupyter_contrib_nbextensions/nbextensions/toc2/main.css

```diff
@@ -4,15 +4,18 @@
 /*background color for links when you mouse over it 
 This style is used on export, but can be overwritted in the livenotebook 
 by selecting a color in the nbextension-configurator*/
-#toc-wrapper li > span:hover {
+
+
+
+#toc-wrapper ul li span:hover {
```

Comment:
this isn't needed `li span` is sufficient & more efficient, since every `li` is in a `ul` anyway, while the `>` prevents nested child spans from being selected (fewer css redraws, more efficient, hopefully)